ksql-engine/src/main/java/io/confluent/ksql/physical/PhysicalPlanBuilder.java

```diff
@@ -190,7 +190,10 @@ private QueryMetadata buildPlanForStructuredOutputNode(final SchemaKStream schem
     }
 
     final QueryId queryId = sinkDataSource.getPersistentQueryId();
-    final String applicationId = addTimeSuffix(serviceId + persistanceQueryPrefix + queryId);
+    String applicationId = serviceId + persistanceQueryPrefix + queryId;
+    if (addUniqueTimeSuffix) {
+      applicationId = addTimeSuffix(applicationId);
+    }
```

Comment:
Actually addUniqueTimeSuffix will always be false here and will always be true for `buildPlanForBareQuery` method. Maybe we should remove this boolean all together.